rfc/rfc-60/rfc-60.md

```diff
@@ -148,7 +147,7 @@ s3://<table_storage_bucket>/0bfb3d6e/<hudi_table_name>/.075f3295-def8-4a42-a927-
 ...
 ```
 
-Note: Storage strategy would only return a storage location instead of a full path. In the above example,
+Storage strategy would only return a storage location instead of a full path. In the above example,
 the storage location is `s3://<table_storage_bucket>/0bfb3d6e/`, and the lower-level folder structure would be appended
 later automatically to get the actual file path. In another word, 
 users would only be able to customize upper-level folder structure (storage location). 
```

Comment:
note: if we don't have any partitioning on the table, then this will achieve random distribution of files across prefixes. partition path is totally optional.